deploy/deploy.sh

```diff
@@ -183,7 +183,7 @@ fi
 
 helm repo add jetstack https://charts.jetstack.io
 helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-helm repo add mariadb-operator https://mmontes11.github.io/mariadb-operator
+helm repo add mariadb-operator https://mariadb-operator.github.io/mariadb-operator
```

Comment:
I think it would be good to add the `--force-update` argument to this command, since it would otherwise fail for users who executed the script before.

```suggestion
helm repo add mariadb-operator https://mariadb-operator.github.io/mariadb-operator --force-update
```